### ISSUE TITLE
Radarr Deleted says movie when its files

### ISSUE THREAD

**Issue by snupple** [Notifiarr/notifiarr#250]
If I delele a file for a movie in Radarr, the notification in Discord will say the entire movie has been deleted:

![image](https://user-images.githubusercontent.com/851790/179162106-0fa0ede7-4883-44c7-85bd-1956bc20ec51.png)

.. but if I do actually delete the movie the notification in Discord says "File(s) for this movie hasbe been deleted:

![image](https://user-images.githubusercontent.com/851790/179162219-3a3881b9-cb65-47c4-bf8f-61d362d579e5.png)

So basically they are the wrong way around :)

**Comment by davidnewhall** [comment]
Thanks for the details, we'll get this fixed!
